src/components/cartItem/cartItem.styles.css

```diff
@@ -0,0 +1,50 @@
+.cart-item-wrapper {
+  background-color: #ffffff;
+  border-radius: 10px;
+  padding: 12px;
+  min-height: 30px;
+  margin-bottom: 20px;
+}
+
+.cart-item-wrapper .cart-item-information {
+  border-right: 2px solid #cdcdcd;
+  position: relative;
+  width: 93%;
```

Comment:
Width 93%? Thats odd.

Reply:
Yes I am trying a other wayout for this.. No more width will be required for this ..

Reply:
Added changes for this removed 93% have used flex now